pages/search.js

```diff
@@ -134,10 +134,11 @@ const Search = ({ appContext }) => {
           content: query.content || 'all',
           group: query.group || 'all',
           source: Object.keys(sourceOptions).includes(query.source) ? query.source : 'all',
+          sort: 'tmdate:desc',
           limit: 1000,
         },
         null,
-        { accessToken, resultsKey: 'notes', sort: 'none' }
+        { accessToken, resultsKey: 'notes', sort: 'tmdate:desc' }
```

Comment:
I wish we can sort them by pdate too, is there a way to decide this in the webfield component?

Reply:
This change is for the general /search page, not the venue homepage search.